quickstarts/microsoft.maps/maps-use-sas/azuredeploy.json

```diff
@@ -0,0 +1,185 @@
+{
```

Comment:
See: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#sort-order-of-properties for the proper sort order of properties

I left a comment in certain spots, but it looks like all of the resources need to be sorted correctly/consistently.

Reply:
Went through and sorted resource property names.